test/python/WMComponent_t/JobAccountant_t/JobAccountant_t.py

```diff
@@ -1373,13 +1374,13 @@ def setupDBFor4GMerge(self):
         self.testRecoMergeWorkflow = Workflow(spec="wf002.xml", owner="Steve",
                                               name="TestRecoMergeWF", task="None")
         self.testRecoMergeWorkflow.create()
-        self.testRecoMergeWorkflow.addOutput("Merged", self.mergedRecoOutputFileset,
+        self.testRecoMergeWorkflow.addOutput("MergedRECO", self.mergedRecoOutputFileset,
```

Comment:
I'm not sure that `MergedRECO` and `MergedAOD` are good definitions since `Merged` is a state, while `RECO` or `AOD` is a data-tier and it is better to keep them separated. Since I do not know where these values will be used I can't comment further, but I suggest do not mix or introduced mixed entities.

Reply:
This is the definition of WMBS output for a workflow.

Reply:
thanks for providing this info, but it does not address issue of mixed entities I raised.

Reply:
I don't know all the details now (of workflow vs fileset vs subscription vs output module definition), but a given output can be either unmerged (carrying part of the output module) or merged (then, by WMCore convention, being named as Merged + output module).

In short, I do not see any issue to be addressed here. In addition, this is a unit test (which was broken btw).